src/AniLink/MorphTo.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import TransitionLink from 'gatsby-plugin-transition-link'
-import { TimelineMax } from 'gsap'
+import TransitionLink from '../'
```

Comment:
@TylerBarnes I've gone ahead and updated this too just as I noticed it, local import rather than from node_modules